packages/react/src/auto/polaris/inputs/relationships/PolarisListMessages.tsx

```diff
@@ -0,0 +1,29 @@
+import { Listbox, Text } from "@shopify/polaris";
+import React from "react";
+
+export const SelectableOption = (props: { label: string; id: string; selected: boolean }) => {
+  const { label, id, selected } = props;
+  return (
+    <Listbox.Option key={id} value={id} selected={selected}>
+      {label}
+    </Listbox.Option>
+  );
+};
+
+export const ListMessage = (props: { message: string }) => (
+  <div style={{ padding: "16px" }}>
+    <Text as="p" tone="subdued">
+      {props.message}
+    </Text>
+  </div>
+);
+
+export const NoRecordsMessage = () => <ListMessage message="No records found" />;
+
+export const getErrorMessage = (errorString: string) => {
+  if (errorString.includes("GGT_PERMISSION_DENIED")) return `Cannot read records from related model`;
```

Comment:
Can we use content here that indicates that it is a permissions error so that a dev knows to go fix that ?

Reply:
Updated the message to indicate that there are missing permissions